lib/iex/lib/iex/introspection.ex

```diff
@@ -265,7 +265,9 @@ defmodule IEx.Introspection do
     _ = case Typespec.beam_types(module) do
       nil   -> nobeam(module)
       []    -> notypes(inspect module)
-      types -> for type <- types, do: print_type(type)
+      types -> for type <- types do
```

Comment:
This highly confusing structure, it'd be better to use `Enum.each(types, & print_type/1)`.
And after all we're not interested in what `for` returning.